resources/lang/de.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Reset Password Notification": "Passwort-Rücksetz-Benachrichtigung",
+  "You are receiving this email because we received a password reset request for your account.": "Sie erhalten diese E-Mail, weil eine Anfrage zum Zurücksetzen des Passworts für Ihr Benutzerkonto gestellt wurde.",
+  "Reset Password": "Passwort zurücksetzen",
+  "This password reset link will expire in :count minutes.": "Der Rücksetz-Link wird in :count Minuten ablaufen.",
```

Comment:
Maybe it would be better to give the user the absolute time, as the email sending and receiving can be delayed

Reply:
This is done by Laravel. I could only reimplement the whole mail notification to make those notifications.